ZenPacks/zenoss/Import4/bin/imp4_check.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
```

Comment:
I think forcing a user to re-run initialize is going to trip them up.  I think instead, we should clear out everythign in staging/zenbackup whenever we run 'check'

One thing to note is that I think you are assuming that every backup will consist of the same exact file names, but this is not true (look at expolded zencatalogservice, for example).